R/server.R

```diff
@@ -0,0 +1,23 @@
+
+#' Start a gRPC server
+#' 
+#' @param impl an implementation of a proto service
+#' @return none
+#' @importFrom methods selectMethod
+#' @useDynLib grpc
+#' @export
+start_server <- function(impl){
```

Comment:
Add port number as a new argument and pass to `grpc_server_add_insecure_http2_port` in `run`?

Reply:
👍